### ISSUE TITLE
Hard to follow documentation

### ISSUE THREAD

**Issue by increpare** [shtrom/Lurch4Adium#18]
Hello - cool work on the plugin!

I and my friend installed the plugin, typed '/carbons on'. I got a 'success' message.  I have no idea the plugin is actually doing anything though.  Is there any visual indication that what we're saying is encrypted?  (or that something's not encrypted).

Some info as to what to expect of a successfully encrypted/unsuccessfully encrypted conversation would be useful for me.

**Comment by shtrom** [comment]
Unfortunately, there is no clear indication this is already discussed in #6, but I haven't had time to follow up on how this could be done.

I'll close this issue as a duplicate of #6.

**Comment by increpare** [comment]
( Ah ok, cool - thanks for the reply :) )
